Artroom1/ArtPieceDB.swift

```diff
@@ -8,6 +8,6 @@
-]
+var arrayOfArtworks: [Artwork] = []
+//    (Artwork(title: "Watching - Erin Nicholls", image: "erin nicholls", attributes: ArtAttributes(mood: .joyful, interest: .tech ))), (Artwork(title: "Fall Scene - John Smith", image: "nowhere", attributes: ArtAttributes(mood: .joyful, interest: .nature))), (Artwork(title: "Mikhail - Annie Leibovitz", image: "Mikhail - Annie Leibovitz", attributes: ArtAttributes(mood: .joyful, interest: .humanity))), (Artwork(title: "Reflections - Erin Nicholls", image: "Reflections_nicholls", attributes: ArtAttributes(mood: .melancholy, interest: .tech))), (Artwork(title: "Norway Night - Sarah Hicks", image: "norway", attributes: ArtAttributes(mood: .melancholy, interest: .nature))), (Artwork(title: "Diagonal of May 25 - Dan Flavin", image: "Diagonal of May 25 - Dan Flavin", attributes: ArtAttributes(mood: .melancholy, interest: .humanity))), (Artwork(title: "Melbourne - Erin Nicholls", image: "melbourne", attributes: ArtAttributes(mood: .meh, interest: .tech))), (Artwork(title: "Isle Saint-Michel - Elger Esser", image: "Isle Saint-Michel - Elger Esser", attributes: ArtAttributes(mood: .meh, interest: .nature))), (Artwork(title: "Studies of Hands - Alexander Maw", image: "Studies of Hands - Alexander Maw", attributes: ArtAttributes(mood: .meh, interest: .humanity))), (Artwork(title: "Poveglia - Elger Esser", image: "Poveglia - Elger Esser", attributes: ArtAttributes(mood: .meh, interest: .nature))), (Artwork(title: "Drifting - Anthony Goicolea", image: "Drifting - Anthony Goicolea", attributes: ArtAttributes(mood: .meh, interest: .nature))), (Artwork(title: "My parents with their grandson, Ross - Annie Leibovitz", image: "My parents with their grandson, Ross - Annie Leibovitz", attributes: ArtAttributes(mood: .joyful, interest: .humanity))), (Artwork(title: "Class Picture - Anthony Goicolea", image: "Class Picture - Anthony Goicolea", attributes: ArtAttributes(mood: .joyful, interest: .humanity))), (Artwork(title: "By Lantern Light - Erin Nicholls", image: "By Lantern Light - Erin Nicholls", attributes: ArtAttributes(mood: .joyful, interest: .humanity)))
```

Comment:
I think you can also get rid of the old data here. Everything is now in your JSON 🙂